Lib/test/test_capi/test_abstract.py

```diff
@@ -1,6 +1,8 @@
 import unittest
 from collections import OrderedDict
-import _testcapi
+from test.support import import_helper
+
+_testcapi = import_helper.import_module('_testcapi')
```

Comment:
I think that such check should be done in `Lib/test/test_capi/__init__.py`. I don't think that it's worth to attempt to run any of sub-tests if _testcapi cannot be imported.

Reply:
I don't agree that it should only be done there, because people can run only a single test file. In case of `./python.exe Lib/test/test_capi/test_WHATEVER.py`, this test must also be skipped correctly.

However, I think that we can *also* check this in `test_capi/__init__.py` just to speed up test runs without `_testcapi`